GNUmakefile

```diff
@@ -1,29 +1,44 @@
-default: fmt lint install generate
+SHELL := /bin/bash
+OUT := $(shell pwd)/_out
 
-build:
-	go build -v ./...
+EXCLUDED_PACKAGES := \
+	github.com/foxboron/terraform-provider-openwrt \
+	github.com/foxboron/terraform-provider-openwrt/internal/api \
+	github.com/foxboron/terraform-provider-openwrt/internal/types \
+	github.com/foxboron/terraform-provider-openwrt/mocks
 
-install: build
-	go install -v ./...
+PACKAGES := $(shell go list ./... | grep -Fvx -f <(printf '%s\n' $(EXCLUDED_PACKAGES)))
+
+.PHONY: clean fmt lint install generate build test snapshot release
+default: clean fmt lint install generate
```

Comment:
`lint` needs to move until after `generate`, as `mocks/` is not created at this point and it will fail.